plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialBrewingStand.java

```diff
@@ -0,0 +1,147 @@
+package com.denizenscript.denizen.objects.properties.material;
+
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.block.data.type.BrewingStand;
+
+import java.util.Set;
+
+public class MaterialBrewingStand implements Property {
+
+    public static boolean describes(ObjectTag material) {
+        return material instanceof MaterialTag
+                && ((MaterialTag) material).hasModernData()
+                && ((MaterialTag) material).getModernData() instanceof BrewingStand;
+    }
+
+    public static MaterialBrewingStand getFrom(ObjectTag _material) {
+        if (!describes(_material)) {
+            return null;
+        }
+        else {
+            return new MaterialBrewingStand((MaterialTag) _material);
+        }
+    }
+
+    public static final String[] handledMechs = new String[] {
+            "bottles"
+    };
+
+    private MaterialBrewingStand(MaterialTag _material) {
+        material = _material;
+    }
+
+    MaterialTag material;
+
+    public static void registerTags() {
+        // <--[tag]
+        // @attribute <MaterialTag.bottles>
+        // @returns ListTag(ElementTag)
+        // @mechanism MaterialTag.bottles
+        // @group properties
+        // @description
+        // Returns a list of booleans that represent whether a slot in a brewing stand has a bottle.
+        // -->
+        PropertyParser.<MaterialBrewingStand, ListTag>registerStaticTag(ListTag.class, "bottles", (attribute, material) -> {
+            return material.getBottleBooleans();
+        });
+
+        // <--[tag]
+        // @attribute <MaterialTag.has_bottle[<#>]>

```

Comment:
Is there a reason that this tag needs to exist? It seems like a redundant duplicate of `bottles.get[x]`

Reply:
Was a leftover from the format being a list of numbers, will remove